.circleci/config.yml

```diff
@@ -491,7 +491,7 @@ jobs:
       - run:
           command: ./gradlew :functional-test:tomcatTest --no-daemon --stacktrace --info --parallel
           environment:
-            TEST_RUNDECK_IMAGE: TOMCAT_TAG:<< parameters.tomcat-vers >>
+            TEST_RUNDECK_IMAGE: << parameters.tomcat-vers >>
```

Comment:
should use an env var that makes sense, it is not a rundeck image, it is a tomcat tag. I think it makes more sense as TOMCAT_TAG

Reply:
i think, instead of an specific name we should only use a generic one, this way we don't have multiple variables with the same purpose